src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,48 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the response json with the
+ * aggregated buildbot data.
+ */
+public class BuildInfo {
+  private final String commitHash;
+  private final String timeStamp;
+  private final String repository;
+  private final String branch;
+
+
+  private final List<Builder> builders;
+
+  public BuildInfo(String commitHash, String timeStamp, String repository,
+                   String branch, List<Builder> builders) {
+    this.commitHash = commitHash;
+    this.timeStamp = timeStamp;
+    this.repository = repository;
+    this.branch = branch;
+    this.builders = new ArrayList<>(builders);
+  }
+
+  public String getCommitHash() {
```

Comment:
[please document these methods](https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required). Focus on the interesting bits that a reader unfamiliar with the project may be interested in: what is a commit hash  and repository? Similarly for the Builder data below.